generator.go

```diff
@@ -412,99 +412,99 @@ func (f *_vfsgen_compressedFile) Read(p []byte) (n int, err error) {
 
-// _vfsgen_file is an opened file instance.
-type _vfsgen_file struct {
-	*_vfsgen_fileInfo
+// vfsgen۰File is an opened file instance.
+type vfsgen۰File struct {
+	*vfsgen۰FileInfo
 	*bytes.Reader
 }
 
-func (f *_vfsgen_file) Close() error {
+func (f *vfsgen۰File) Close() error {
 	return nil
 }
 {{end}}
-// _vfsgen_dirInfo is a static definition of a directory.
-type _vfsgen_dirInfo struct {
+// vfsgen۰DirInfo is a static definition of a directory.
+type vfsgen۰DirInfo struct {
 	name    string
 	modTime time.Time
 	entries []os.FileInfo
 }
 
-func (d *_vfsgen_dirInfo) Read([]byte) (int, error) {
+func (d *vfsgen۰DirInfo) Read([]byte) (int, error) {
 	return 0, fmt.Errorf("cannot Read from directory %s", d.name)
 }
-func (d *_vfsgen_dirInfo) Close() error               { return nil }
-func (d *_vfsgen_dirInfo) Stat() (os.FileInfo, error) { return d, nil }
-
-func (d *_vfsgen_dirInfo) Name() string       { return d.name }
-func (d *_vfsgen_dirInfo) Size() int64        { return 0 }
-func (d *_vfsgen_dirInfo) Mode() os.FileMode  { return 0755 | os.ModeDir }
-func (d *_vfsgen_dirInfo) ModTime() time.Time { return d.modTime }
-func (d *_vfsgen_dirInfo) IsDir() bool        { return true }
-func (d *_vfsgen_dirInfo) Sys() interface{}   { return nil }
-
-// _vfsgen_dir is an opened dir instance.
-type _vfsgen_dir struct {
-	*_vfsgen_dirInfo
+func (d *vfsgen۰DirInfo) Close() error               { return nil }
+func (d *vfsgen۰DirInfo) Stat() (os.FileInfo, error) { return d, nil }
+
+func (d *vfsgen۰DirInfo) Name() string       { return d.name }
+func (d *vfsgen۰DirInfo) Size() int64        { return 0 }
+func (d *vfsgen۰DirInfo) Mode() os.FileMode  { return 0755 | os.ModeDir }
+func (d *vfsgen۰DirInfo) ModTime() time.Time { return d.modTime }
+func (d *vfsgen۰DirInfo) IsDir() bool        { return true }
+func (d *vfsgen۰DirInfo) Sys() interface{}   { return nil }
+
+// vfsgen۰Dir is an opened dir instance.
+type vfsgen۰Dir struct {
+	*vfsgen۰DirInfo
 	pos int // Position within entries for Seek and Readdir.
 }
 
-func (d *_vfsgen_dir) Seek(offset int64, whence int) (int64, error) {
+func (d *vfsgen۰Dir) Seek(offset int64, whence int) (int64, error) {
 	if offset == 0 && whence == io.SeekStart {
 		d.pos = 0
 		return 0, nil
 	}
-	return 0, fmt.Errorf("unsupported Seek in directory %s", d._vfsgen_dirInfo.name)
+	return 0, fmt.Errorf("unsupported Seek in directory %s", d.name)
 }
 
-func (d *_vfsgen_dir) Readdir(count int) ([]os.FileInfo, error) {
-	if d.pos >= len(d._vfsgen_dirInfo.entries) && count > 0 {
+func (d *vfsgen۰Dir) Readdir(count int) ([]os.FileInfo, error) {
+	if d.pos >= len(d.entries) && count > 0 {
 		return nil, io.EOF
 	}
-	if count <= 0 || count > len(d._vfsgen_dirInfo.entries)-d.pos {
-		count = len(d._vfsgen_dirInfo.entries) - d.pos
+	if count <= 0 || count > len(d.entries)-d.pos {
+		count = len(d.entries) - d.pos
 	}
-	e := d._vfsgen_dirInfo.entries[d.pos : d.pos+count]
+	e := d.entries[d.pos : d.pos+count]
```

Comment:
Yes, relatively unrelated. I could've done this before too.

I got rid of it because I thought it also increases readability.